engine/src/flutter/lib/web_ui/test/engine/semantics/semantics_test.dart

```diff
@@ -3629,6 +3632,92 @@ void _testTabs() {
   });
 }
 
+void _testTables() {
+  test('nodes with table role', () {
+    semantics()
+      ..debugOverrideTimestampFunction(() => _testTime)
+      ..semanticsEnabled = true;
+
+    SemanticsObject pumpSemantics() {
+      final SemanticsTester tester = SemanticsTester(owner());
+      tester.updateNode(
+        id: 0,
+        role: ui.SemanticsRole.table,
+        rect: const ui.Rect.fromLTRB(0, 0, 100, 50),
+      );
+      tester.apply();
+      return tester.getSemanticsObject(0);
+    }
+
+    final SemanticsObject object = pumpSemantics();
+    expect(object.semanticRole?.kind, EngineSemanticsRole.table);
+    expect(object.element.getAttribute('role'), 'table');
+  });
```

Comment:
It's been so long I don't remember why we do this anymore, but all other tests end with:

```
    semantics().semanticsEnabled = false;
```

Maybe because it's the only way to clear the tree from the previous test?